### PR TITLE
feat: pointer event toggle highlight

### PR DESCRIPTION
**Pull request by pravusjif**
* Implements support for new `PointerEvents` `showHighlight` property
* Fixed bug existent bug for the case of using `pointerEventsSystem.onPointerDown()` without any of the pointer event optional properties (those entities weren't clickable before this PR)

Related PRs:
- https://github.com/decentraland/protocol/pull/224
- https://github.com/decentraland/unity-explorer/pull/2651

### CODE REVIEW

**Comment by pravusjif** [comment]
I'll update the protocol package to point to the latest published one after the protocol PR is merged.

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "bugs": "https://github.com/decentraland/js-sdk-toolchain/issues",
   "dependencies": {
     "@actions/core": "^1.10.0",
-    "@dcl/protocol": "1.0.0-11406954347.commit-ba19c4f",
+    "@dcl/protocol": "https://sdk-team-cdn.decentraland.org/@dcl/protocol/branch//dcl-protocol-1.0.0-11593429245.commit-6dd6863.tgz",
```
Comment (gonpombo8) [inline package.json:9]: update

File: packages/@dcl/ecs/src/systems/events.ts [hunk 2]
```diff
@@ -110,14 +111,15 @@ export function createPointerEventsSystem(engine: IEngine, inputSystem: IInputSy
   }
 
   function setPointerEvent(entity: Entity, type: PointerEventType, opts: EventSystemOptions) {
-    if (opts.hoverText || opts.showFeedback) {
+    if (opts.hoverText || opts.showFeedback || opts.showHighlight) {
```
Comment (gonpombo8) [inline packages/@dcl/ecs/src/systems/events.ts:114]: is this necessary ? not sure why we have this so asking just in case

Comment (pravusjif) [reply packages/@dcl/ecs/src/systems/events.ts]: great catch! by removing that check we solve an existent bug (when using `pointerEventsSystem.onPointerDown()` without any of the pointer event optional properties)

File: package.json (hunk 1 above)
Comment (pravusjif) [reply package.json]: Done!

File: packages/@dcl/ecs/src/systems/events.ts (hunk 2 above)
Comment (pravusjif) [reply packages/@dcl/ecs/src/systems/events.ts]: tackled at https://github.com/decentraland/js-sdk-toolchain/pull/1024/commits/0abea71304fb5c50ecd1ebbfc4af91a83673cd83

**Review by gonpombo8 — APPROVED** [review]

**Pull request merged**